util/fipstools/delocate/delocate.go

```diff
@@ -488,6 +490,19 @@ func (d *delocation) processAarch64Instruction(statement, instruction *node32) (
 		}
 
 		return d.loadAarch64Address(statement, targetReg, symbol, offset)
+	case "bl":
+		// We were relying on symbols defined with ".comm" to populate bssAccessorsNeeded,
+		// but gcc release does not use ".comm" to define common symbols. The accessor
```

Comment:
* In this comment and the following one, we say `gcc release`, do we need to be more specific about gcc-7, or this was also the case with gcc-11 (or other versions)?
* Would a `release build by gcc` be clearer than `gcc release`?
* I think the PR description and the commit message should mention that these issues were not occurring in the debug build with gcc.
* Is it correct to modify "The accessor functions" to "The symbols requiring accessor functions (i.e. with a suffix`_bss_get`)"